com.unity.perception/Runtime/GroundTruth/PerceptionCamera.cs

```diff
@@ -286,5 +306,12 @@ public bool RemoveLabeler(CameraLabeler cameraLabeler)
             }
             return false;
         }
+
+#if URP_PRESENT
```

Comment:
Sort of mirroring the above point - having a function of a class per an ifdef will result in inflexible arch down the road.  I think it'd be better to at least put the IFDEF inside and then call up to the parent explicitly if this is an override. 

Does C# have an explicit override keyword like C++?  If so I think that could also make this a bit more clear

Reply:
Fixed. This isn't overriding anything in the base class though, so I think it doesn't apply.